ckanext/switzerland/plugins.py

```diff
@@ -73,6 +73,7 @@ def get_validators(self):
     def dataset_facets(self, facets_dict, package_type):
         lang_code = toolkit.request.environ['CKAN_LANG']
         facets_dict = collections.OrderedDict()
+        facets_dict['private'] = plugins.toolkit._('Draft')
```

Comment:
Maybe using `Drafts` would be a little more consistent? Other than that this PR looks good to me :)

Reply:
I felt that `Draft` was more logical since that's the state that the facet is filtering on, with the boolean true/false options. E.g. you can have "the category Admin" or "the political level Confederation", but you can't really have "the draft True".

I can change it if you disagree, though.